xtext-website/_posts/releasenotes/2021-12-31-version-2-26-0.md

```diff
@@ -21,6 +21,7 @@ TODO
 
 ## Enhancements
 
+- [xtext-maven#139](https://github.com/eclipse/xtext-maven/issues/139) : xtext-maven now automatically installs trace information for DSLs with a Java target.
```

Comment:
wonder if we should be more precise here about xbase usage

Reply:
Basically you could write a generator that produces java and not use xbase. But I could mention that as an example.

Reply:
yes and no. You wont have tracing information unless you make use of the TracingSugar / generateTracedFile feature.
(which 99% of the generators that produce java i have seen dont use)

Reply:
OK, i'll mention Xbase. People using custom java generators and manually using trace options should know what they're doing anyway.